### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by caldav**
Some ordering and content refresh in CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,54 +1,65 @@
-# Contributing
+# How to contribute
 
-Issues, content and code contributions are all very appreciated. We ask that you read the relevant sections of this document before contributing.
+Issues, tutorials requests, new tutorials, content fixes and code contributions are all very appreciated. We ask that you read the relevant sections of this document before contributing.
```
Comment (degville) [inline CONTRIBUTING.md]: very *much* appreciated?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,54 +1,65 @@
...
 
+## Contributing a content fix
 
-## Issues
+Have you spotted a typo? A screenshot that needs updating? A nonsensical sentence? While we strive to foster and maintain quality content, mistakes happen. If you have the time, please open a new Pull Request, propose a fix and add the "Tutorials content" label to it, it will be reviewed quickly.
```
Comment (degville) [inline CONTRIBUTING.md]: super-minor, but the label has a capital C - Tutorials Content.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,54 +1,65 @@
...
-Please try to create simple commits with readable commit messages. Do split the work into smaller pull requests, rather than one big pull request, unless you are proposing a new tutorial.
-
-Some sections to include when creating a PR:
- - Short summary of the work completed
- - Steps to view and test visual or code changes
- - Links to any relevant issues
- - Screenshots to demonstrate the work
+See our [writing guide](https://tutorials.ubuntu.com/?q=tutorial+guidelines) for extensive documentation on writing and publishing a new tutorial. It contains step-by-step documentation, content guidelines and examples to help you make the best of this platform. 
 
+All tutorials are reviewed and merged by the Canonical Docs team in order to maintain content consistency. You can get in touch with us on Freenode IRC, in #ubuntu-doc.
```
Comment (degville) [inline CONTRIBUTING.md]: this is just a suggestion, but I think it helps link to Freenode's webchat when referencing IRC, as readers may be unfamiliar with it, i.e: http://webchat.freenode.net/?channels=ubuntu-doc

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,54 +1,65 @@
...
+#### What's the format of a tutorial?
 
-### Markdown
+Each tutorial is a single Markdown file, placed inside the `tutorials` folder of this repository.
 
-_If you find any bugs in Markdown rendering, please report them to [Ubuntu tutorial deployment repository](https://github.com/ubuntu/tutorial-deployment)_
+#### Where are images hosted?
 
-Our [Ubuntu tutorial guidelines](examples/example-tutorial.md) markdown tutorial is inside the `examples` folder. This contains useful information and resources for creating new tutorials, including documentation on custom markdown tags used in this project. You can view this tutorial in full by running the local server and can use the source as a template to create new content.
+Image can be placed inside this repository, alongside a tutorial in an `images` folder.
```
Comment (degville) [inline CONTRIBUTING.md]: *Images* can be placed

**Comment by caldav** [comment]
@degville Thanks for the review, I addressed all comments.

**Pull request merged**